app/scripts/proactive/config.js

```diff
@@ -62,6 +62,50 @@ define(function () {
             'Script/flow' : [" ", "scalaw", "groovy", "javascript", "python", "cpython", "ruby", "powershell", "R"],
             'Script/clean' : [" ", "bash", "cmd", "kubernetes", "docker-compose", "scalaw", "groovy", "javascript", "python", "cpython", "ruby", "perl", "powershell", "R"]
         },
+        extensions_to_languages: {
+            "sc": "scalaw",
+            "scala": "scalaw",
+            "groovy": "groovy",
+            "js": "javascript",
+            "py" : "python",
```

Comment:
If I remembered correctly, py is the extension for jython required by jsr223, and we created a new extension which is called cpy for python. Need to be checked.